frontend/scss/components/atoms/headline.scss

```diff
@@ -21,7 +21,7 @@ h1 {
 			margin-top: 0;
 
 			@media (min-width: 768px) {
-				margin-bottom: 2rem;
+				margin-bottom: 0.75rem;
```

Comment:
Leave this alone and rather use negative margins to pull the pills back up if they are present.